src/runtimes/clients/AutoConfigurableDockerClient.ts

```diff
@@ -17,5 +20,15 @@ export class AutoConfigurableDockerClient extends DockerClient implements AutoCo
         const config = vscode.workspace.getConfiguration('docker');
         const dockerCommand = config.get<string | undefined>('dockerPath') || 'docker';
         this.commandName = dockerCommand;
+
+        if (ext.outputChannel.debugLoggingEnabled && isLinux()) {

```

Comment:
Can this go in with the other debug-only code?

Reply:
Refactored this to use a method from the new diagnostics utility functions